unit_test/tstParticleInit.hpp

```diff
@@ -28,9 +28,12 @@ using namespace Picasso;
 namespace Test
```

Comment:
I think we can remove this test completely

Reply:
removed